monitoring/ceph-mixin/tests_alerts/test_alerts.yml

```diff
@@ -1886,3 +1886,229 @@ tests:
+       eval_time: 45m
+       exp_samples:
+         - labels: '{ceph_daemon="client.admin.40628"}'
+           value: 3
+     # query 2
+     - expr: sum by (ceph_daemon) (ceph_rbd_mirror_snapshot_snapshots) * .1
+       eval_time: 45m
+       exp_samples:
+         - labels: '{ceph_daemon="client.admin.40628"}'
+           value: 0.2
+   # prometheus alert test
+   alert_rule_test:
+     # negative test
+     - eval_time: 2m
+       alertname: CephRBDMirrorImagesNotInSyncVeryHigh
+       exp_alerts:
+      # positive test where alert is fired
+     - eval_time: 50m
+       alertname: CephRBDMirrorImagesNotInSyncVeryHigh
+       exp_alerts:
+         - exp_labels:
+             ceph_daemon: "client.admin.40628"
+             oid: "1.3.6.1.4.1.50495.1.2.1.10.4"
+             severity: "critical"
+             type: "ceph_default"
+           exp_annotations:
+             description: "Very high no: of unsynced images detected."
+             summary: "No: of unsynced images are very high."
+
+ # alert: "CephRBDMirrorImageTransferBandwidthHigh"
+ - interval: 1m
+   input_series:
+     - series: 'ceph_rbd_mirror_journal_replay_bytes{ceph_daemon="client.admin.40628"}'
+       values: '0+0x10 1+0x5 10+30x25 736+200x30'
+   # prometheus query test
+   promql_expr_test:
+     # test each couple of rates
+     # rate 1
+     - expr: rate(ceph_rbd_mirror_journal_replay_bytes[5m])
+       eval_time: 5m
+       exp_samples:
+         - labels: '{ceph_daemon="client.admin.40628"}'
+           value: 0.0
+     # rate 2
+     - expr: rate(ceph_rbd_mirror_journal_replay_bytes[5m])
+       eval_time: 20m
+       exp_samples:
+         - labels: '{ceph_daemon="client.admin.40628"}'
+           value: 0.33
+     # rate 3
+     - expr: rate(ceph_rbd_mirror_journal_replay_bytes[5m])
+       eval_time: 40m
+       exp_samples:
+         - labels: '{ceph_daemon="client.admin.40628"}'
+           value: 0.5
+     # rate 4
+     - expr: rate(ceph_rbd_mirror_journal_replay_bytes[5m])
+       eval_time: 50m
+       exp_samples:
+         - labels: '{ceph_daemon="client.admin.40628"}'
+           value: 3.3333333333333335
+   # prometheus alert test
+   alert_rule_test:
+     # negative test
+     - eval_time: 2m
+       alertname: CephRBDMirrorImageTransferBandwidthHigh
+       exp_alerts:
+      # positive test where alert is fired
+     - eval_time: 50m
+       alertname: CephRBDMirrorImageTransferBandwidthHigh
+       exp_alerts:
+       - exp_labels:
+           ceph_daemon: "client.admin.40628"
+           oid: "1.3.6.1.4.1.50495.1.2.1.10.5"
+           severity: "warning"
+           type: "ceph_default"
+         exp_annotations:
+           description: "The amount of information replicated from remote cluster has been heavy increased in the last 5 minutes"
+           summary: "A very high network usage detected due to image transfers. Please reduce the no: of images to replicate."
```

Comment:
can we provide `image count` for users to reduce, instead of just saying `no:`

Reply:
Number of `image count` value, that is to be reduced, is a dynamic one, depending on the cluster and network bandwidth etc, it changes.
`description` and `summary` are all static texts.